src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -252,13 +252,13 @@ private string CreateTokenPrivate(JObject payload, SigningCredentials signingCre
             {
                 var now = DateTime.UtcNow;
```

Comment:
We could be slightly better performance by calculating long now = EpochTime.GetIntDate(DateTime.UtcNow).
We could also add a method EpochTime.UtcNow that is tuned for perf.

Reply:
That's a good point! I changed this so that EpochTime.GetIntDate(DateTime.UtcNow) is only called once.

What do you mean by 'tuned for perf'?

Reply:
:shipit:

Reply:
@mafurman have a look at: brentsch/epochtime for an idea of how we can improve perf.